src/pages/about-us/orbit-blog.tsx

```diff
@@ -84,7 +84,7 @@ export const getServerSideProps: GetServerSideProps = async (
   })
   // Calculate total pages
   // If NaN, return 1 page
-  const totalPages = Math.ceil(articlesCount / articlesPerPage) || 1
+  const totalPages = Math.floor(articlesCount / articlesPerPage) || 1
```

Comment:
This determines the number of pages to display in the `Pagination` component. If the number of published articles was exactly 10 (the number of articles per page, specified in a constant on line 72) `Math.ceil` was causing the calculation to round up, creating a page link in the component that just linked to an empty page. Changing this to `Math.floor` returns the greatest integer less than or equal to its numeric argument, fixing the issue.

Reply:
Suggestion: I did some testing this seems to not be an issue because of `Math.ceil` but the fact that the graphql query `articlesCount` near line 75 is returning the total number of articles, regardless of their status. I had 12 in my db and page 2 didn't show up. I marked them as draft and the total count was still 12. So I've pushed a commit up that updates the query filter things down to just published and puts `Math.ceil` back in since with 12 articles we want 2 pages and 12/10 is 1.2.

Reply:
Probably worth eventually adding an e2e test for this, unless there are some orbit-blog unit tests around. Is this something we can test in `src/__tests__/pages/about-us/orbit-blog.test.tsx`?

Reply:
Good catch on the `articlesCount`! I'm not seeing any orbit-blog tests anywhere, but I agree we should test somehow. I can add that test file for the orbit-blog and mock out the query like you mentioned. Do you want me to also create a story in the backlog for an e2e test for pagination?

Reply:
Bummer, I think an e2e test would be good enough especially since that will test a real query. Though I would want to wait for the seed database work to be done before starting it since we'll need enough data to show the pagination. Long way to say yes please add a story to create an e2e test for this.